install_files/ansible-base/roles/ossec/tasks/configure_server.yml

```diff
@@ -21,11 +25,13 @@
   command: >
     gpg
     --homedir /var/ossec/.gnupg
-    --import /var/ossec/{{ ossec_alert_gpg_public_key }}
+    --import /var/ossec/{{ item }}
   become: yes
   become_user: "{{ ossec_group }}"
-  register: add_ossec_gpg_key_result
-  changed_when: "'imported: 1' in add_ossec_gpg_key_result.stderr"
```

Comment:
I think you can preserve these two lines such that this task is idempotent

Reply:
right. I removed them because I was not sure about the precedence of when/with_items compared to changed_when. Done